src/database/database-dummy.cpp

```diff
@@ -112,6 +138,16 @@ bool Database_Dummy::removeModEntry(const std::string &modname, const std::strin
 	return false;
 }
 
+bool Database_Dummy::removeModEntries(const std::string &modname)
+{
+	auto mod_pair = m_mod_meta_database.find(modname);
+	if (mod_pair != m_mod_meta_database.end() && !mod_pair->second.empty()) {
```

Comment:
is returning false when already empty part of the interface contract?

Reply:
Yes, the method returns false when the implementation is sure nothing changed, just like `removeModEntry`.